src/main/java/seedu/club/logic/commands/Command.java

```diff
@@ -65,7 +65,8 @@ protected void requireToSignUp () throws CommandException {
      * Requires exco access to use the command
      */
     protected void requireExcoLogIn () throws CommandException {
-        if (!model.getLoggedInMember().getGroup().groupName.equalsIgnoreCase(Group.GROUP_EXCO)) {
+        boolean isExco = model.getLoggedInMember().getGroup().toString().equalsIgnoreCase(Group.GROUP_EXCO);
+        if (!model.getLoggedInMember().getGroup().toString().equalsIgnoreCase(Group.GROUP_EXCO)) {
```

Comment:
why create boolean isExco then not using it?

Reply:
oops that was for debugging. i'll remove that 👍

Reply:
i just used `isExco` in the if condition